labrad/test/test_signal.py

```diff
@@ -0,0 +1,66 @@
+import Queue
```

Comment:
This is a standard library module: https://docs.python.org/2/library/queue.html

I think it was renamed to `queue` in python 3 when they normalized names and capitalization throughout the standard library.